system/Common.php

```diff
@@ -563,6 +564,26 @@ function function_usable(string $functionName): bool
     }
 }
 
+if (! function_exists('htmltag_endline')) {
```

Comment:
`Common.php` is already quite large and I would like to add as few functions as possible.
This function is not common. Only form/html helper uses it. So it should not be added here.

What about adding a class and a static method?
Or is there some other appropriate place?
@codeigniter4/core-team

Reply:
I'm okay with adding this to the Common.php. It seems to be well-aligned with what we usually do.

I would definitely add a static variable inside this function.

Adding an extra class just to handle this case seems odd. Are we expecting to have more use cases like this? Maybe something like `config('DocTypes')->htmlTagEndline()`? But I'm unsure if I want this type of call in the helper.